pvlib/spectrum/mismatch.py

```diff
@@ -235,3 +235,158 @@ def integrate(e):
         smm = pd.Series(smm, index=e_sun.index)
 
     return smm
+
+
+def martin_ruiz(clearness_index, airmass_absolute, module_type=None,
+                model_parameters=None):
+    r"""
+    Calculate spectral mismatch modifiers for POA direct, sky diffuse and
+    ground diffuse irradiances using the clearness index and the absolute
+    airmass.
+
+    .. warning::
+        Included model parameters for ``monosi``, ``polysi`` and ``asi`` were
+        estimated using the airmass model ``kasten1966`` [1]_. It is heavily
+        recommended to use the same model in order to not introduce errors.
+        See :py:func:`~pvlib.atmosphere.get_relative_airmass`.
+
+    Parameters
+    ----------
+    clearness_index : numeric
+        Clearness index of the sky.
+
+    airmass_absolute : numeric
+        Absolute airmass. Give attention to algorithm used (``kasten1966`` is
+        recommended for default parameters of ``monosi``, ``polysi`` and
+        ``asi``, see [1]_).
```

Comment:
If the model requires a specific air mass calculation, then perhaps it is better to have zenith angle as input.